mmseg/models/backbones/fast_scnn.py

```diff
@@ -0,0 +1,361 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, constant_init, kaiming_init
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmseg.models.backbones.mobile_net_v2 import InvertedResidual
+from mmseg.models.decode_heads.psp_head import PPM
+from mmseg.ops import DepthwiseSeparableConvModule, resize
+from ..builder import BACKBONES
+
+
+class LearningToDownsample(nn.Module):
+    """Learning to downsample module.
+    Args:
+        in_channels (int): Number of input channels.
+
+        dw_channels1 (int): Number of output channels of the first
+            depthwise conv (dwconv) layer.
+
+        dw_channels2 (int): Number of output channels of the second
+            dwconv layer.
+
+        out_channels (int): Number of output channels of the whole
+            'learning to downsample' module.
+
+
+        conv_cfg (dict|None): Config of conv layers.
+        norm_cfg (dict|None): Config of norm layers.
+        act_cfg (dict): Config of activation layers.
+    """
```

Comment:
No newline is needed between arguments.
Default value is missing.

Reply:
Fixed.